sdk/attestation/azure-security-attestation/test/samples/attestation/attestation.cpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief This sample provides the code implementation to use the Key Vault Certificates SDK client
+ * for C++ to create, get, update, delete and purge a certificate.
+ *
+ * @remark The following environment variables must be set before running the sample.
+ * - ATTESTATION_AAD_URL:  Points to an Attestation Service Instance in AAD mode.
+ * - ATTESTATION_ISOLATED_URL:  Points to an Attestation Service Instance in Isolated mode.
+ * - LOCATION_SHORT_NAME:  Specifies the short name of an Azure region to use for shared mode
+ * operations.
+ * - AZURE_TENANT_ID:     Tenant ID for the Azure account. Used for authenticated calls to the
+ * attestation service.
+ * - AZURE_CLIENT_ID:     The Client ID to authenticate the request. Used for authenticated calls to
+ * the attestation service.
+ * - AZURE_CLIENT_ID:     The Client ID to authenticate the request. Used for authenticated calls to
+ * the attestation service.
```

Comment:
were these used in the sample?

Reply:
They will be in other samples.

Reply:
Should we document only the env vars required for each sample?  

This sample is asking for env vars that are not consumed/used

Reply:
There will be code which consumes them after I implement Tpm attestation in the client. I don't want to remove them now only to add them later.